index.html

```diff
@@ -0,0 +1,20 @@
WS3NbWN+973mOxvPXvtK4IKuH+9Hx0ruO6fmH0nkFNr9O0CNjY7awkqeYE2ialcamVXBvt7wOnwxVgCpswG50J7SR67i90YgcwNZzmGxhXc2nRYXj5yhSBbn3EDW4Zj6LhA9FS9wAWpqSe951XCuBgYmpiCAAwUILbeUAn9Jm+HMZSrMqhfPpoBp6k8p31TDCwKixG8DMca6xFbw6iaxLAhyEyRF7xyIlvbpANQJr0th6CZOWalBroPkfaBNlvGy2iRoRHSABWM36Q22gkQI8seH90IoGO9OVqtPtNJl1kWT/O0DIq4S8qvhO03GoyI0O8DAbB7yP/s8zeFDtHWl1EDETA305nT9Z1nDKa00CIu252ueZlXB4Y5wbaajbabNOmBtAkpAm95OiwFFpKg0gLLBy6wrxCABHpIaThtuRO8stGKgesCpisOHRkbZgVPPQi0+ccTgTSrVaTCxpuy/I6T6YYTgPHng770nE0ABUGrqPzga3/mgePV/L8WvQDf3lWtVLC2th7TY/CXJvvz9f2gVcMBoADAqcM4dnIz3t+pHQTdxuCRqJWmgVhqDz0kGGULYDTrN/D8ODqMr2Pe2sDiuEcYq4aoHUm4IDDkRfX3n0ZwbGrVpU6im6uisPcAieG8Z4AEBYkGdx9k3GAyNhSfMnmW/8G1h6GB3mNwmbzKNfrM3IV0Ok6IiBVpAjUXgYFo806vDhbykj12mbVQqdRaAg3KWsNVsbcj/AJeVEaWUw91Hr+kC2N9JZWRog2tbvJUFoBZZG4vJLiA0BskUK0UDOEYxwYxMAXglOUNjeOOkCMrHpULmO5ABJ07mWMMQRmB3/WBOqgC0NOkAwqe+vKBOFEcNAUwlI/zrANR3jRvlE7dIDwWJjX/vHIgODpAIhW2gtA858eeEtWxNAd6iAf8AWv7icLTwx01B+s9/caW3E8u8YcHahVDIP+FUOgF/K3Mdgd4GTwnCLe7a9uk3amBTcC3pppMrCF9CRpbT0vNaniUXRiAe8DD43h/IeQ9ZF9lllx7DrTe3sRL/ABtAymxFpzPAcccPjKNS9gHyt0ytofqIH0Ko0jyCnUBGnOSI0BEStWohhY+0sMb6yL0gZTUcpsf9xLdMSatTuOhkKnrAtIYYMgR4WeBMYDxg8V/nAe/eKNcxQKYMe0SCK/rAAqIWX0jH9IjpA5nxlWYmjQViDVfKbblec63D0gqBQALAD5Cc7i8L95j6GlxTpu5PRico166mdNfW0BZ77Q0mdxLilPDoGqMFuwVRzZibBVHMkkTQpEmBIIZkQ6R7wCtrCkRMZGvrAJG6+0kOkjYXjjWAZjAQiIIMASJBj8ClVGpuLhh8u47iWiIN4HjvEOE1qNVqTFrDzBgPiW+hB5ektYeiLXIJ7mejcb4WK6FT8Q1U9+nvPP6hKeQizA2I9OUCLGICpB7GcLxqmAzW+c7OvX0O/vOP4tqT3vA9+4H/AOno31P3ab/yiXhM/ghvh6RH/LT/APIl1BAlvIoapAddRAINKuJsD6yyUlLibhELtYBdSTyEAhVEenUB22nmfHfHecmnh9BsX0v/AKYHBeIvSXR2a9jrrvvA9UVh02hqZyeB8QubZ1JnS4asHUMpuD+kCxeKPligVE/WOdYgY3pAQEEwjaCbQMngJdsTinfZXWmvoq3v/wBU2sRXCC59hFSpBbkWFzmNuvUzMr1C7dht+94HHccwr1+KYJmJKAlsv5Rk817T01BMnA4BQ4cgFgCAel95sKID5Y5Ea8YQAqch1mJ4p8RpgaaO4LBnVLDex3IHO282hqb+05nivB1xWOpZxdMMufKdmqOfJcdgpPuIHS4PELURXXZgCNLaEcxLGWVytjcW7yam3ygSgxMsE6RB4D35Rgu8T6wc1oDtOK8Y8Os6110vZXG3SzGdspuLSnj8IHR0P5hbX9DA8sxdO2Y+04/iq7+86XHYnI7o51UlSOhvOW4o9ydYHvPhJy2Ew5O5pp9LfsJrBdd5zP2fYrPgaBO4TKf9JInUFYD+9oK7AmGRBbtAGVOK4MVqb0m2dSvzFpbEFl/tA+Xq1VqLuhUlkYqfbSdFwTj6kBWTzDaS/aJwFkx9Rx8NS1QW9AGHzH6zAwFPJWQ/+5b/ADgdJi/G4RsqJmPO5+dp6D4L8TU8SgyAqwIDqeV72InmviLwq61WdRmVrMCOh5Trvs44I+Hd6jkBXQf6cpvA9O94pk/95ML/AM5P/sIoF0a6xowOkZoB3gxLGKwBxT2TKOcgwtOS4nU27CT0Et3gT0Rb0k0iBktzaAs3eBUeJlgZdYBjlPP+EeKP/F8Vh3IyOUVCf46aC6+9z7gzv2OnYT5s4hii2KrVlJDfes6sNwQxsf0gfSphodJzng3xAuLw6uPjXyuvRx+x3950SHnAIWG8YmO407wFUwCUx2Ag5Y6nnADUSUax7AwdtIHkn2m8JanXFZL5ao1sNnX+u889qudQZ9C+LeFfiMM6DVwMyfzAfvPnvG0yGIbQ3IIOmvSB6z9j+JzYV0v/AOXUPybWeiltp4/9imJ/4mJp33VHA92B/aewLAdmlWrW17W1lsLI6qCAKHQRnOsJYJGsDy77V9K1A8ijj5Ef1nmuJxWWoptexH6c56V9sbW/DdfPr28s8pRLt5jA9EwPiJq6quTVQASDe49J0GBxLJQxHQU2sPb+88xwdJkH3lJwSNxre3cc56zwGgKqBTs6An3GsDzml4Wq1QKiobNqN4p7thsKiIqKoAUADSKBBEYooBLCEUUCIyym3tFFAlXl6QxFFAapsZWG3vFFAWO+B/5T9J8xp+b+ZvqYooHoX2QOfxVUXNsh05bjlPY6e0UUA1jmKKAzbxliigPzEaKKAQlLi+EptTfMiHQ7qDy7iKKBx/2VYdBSrEKoNzqAAfiPOd8v9YooB8hBeKKBHyjPtFFA8i+2j48N6P8AVJ5wu4jRQNSl8DeonrPgr8n/AMYjxQOzWKKKB//Z" alt="" id="image">
```

Comment:
-Kindly Download the image instead and use an img tag

Reply:
- Good job phrasing your comment politely but you have to also provide a reason why you are requesting this particular change.